### DISCUSSION TITLE
Turn off back window led ROG label light for ROG G513

### DISCUSSION THREAD

**Discussion by IsoMan909** [seerge/g-helper#2924] [Ideas]
I suggest to add an option to turn on / off ROG label light, just like the light bar option in the extra tab.
ASUS ROG Strix G15 G513RM, Ryzen 7 6800H , RTX 3060.
I can't find this option in the latest version.

**Comment by seerge** [comment]
@IsoMan909 hello, what specifically do you mean under ROG label ? If this is a logo - you should have a `Logo` toggle under `Extra`.

**Reply by IsoMan909** [reply]
yes sir, there is no Logo toggle under Extra.

**Reply by seerge** [reply]
@IsoMan909 but does that option exist in Armoury?

**Reply by IsoMan909** [reply]
yes, I was disabling it in Armoury and it was working ok. but since I used G-Helper this option doesn't exist.

**Reply by seerge** [reply]
@IsoMan909 just check "Per key RGB" checkbox under `Extra` and restart app, it should appear

**Reply by IsoMan909** [reply]
I did that but nothing changed.
<img width="371" alt="image" src="https://github.com/user-attachments/assets/e9059c1f-e08f-4db4-938e-e9806bdd53c9">

**Reply by seerge** [reply]
@IsoMan909 try this build

[GHelper.zip](https://github.com/user-attachments/files/16508846/GHelper.zip)

**Reply by IsoMan909** [reply]
It now appears in the application, but not making any effect when enabling or disabling it.

**Reply by seerge** [reply]
@IsoMan909 in this build all possible checkboxes should be visible, try if Logo or Lid checkbox for Awake / Battery status turns it ON or OFF 
[GHelper.zip](https://github.com/user-attachments/files/16513141/GHelper.zip)

**Reply by IsoMan909** [reply]
I tried to disable all of them but it's still ON. 
<img width="376" alt="image" src="https://github.com/user-attachments/assets/f10cb3bf-6f52-473c-8652-c626316ac447">

**Reply by seerge** [reply]
@IsoMan909 then even if it's possible to control it, it's done in some unorthodox way. The only way to discover that is to install AC again, and intercept USB commands that it sends. Let me know if you are willing to do that.

---

Install https://www.hhdsoftware.com/device-monitoring-studio (will require reboot)

1. Launch Monitoring Studio, pick Asus device from the list and double click on it
![Screenshot 2024-01-20 163125](https://github.com/seerge/g-helper/assets/5920850/e9a28802-08cb-423c-812b-7b526b051a5f)

2. Add ``Packet View`` to Procesing and click ``Start``
![Screenshot 2024-01-20 163136](https://github.com/seerge/g-helper/assets/5920850/354340f0-2fcb-473b-a49f-2e01b25d00a9)

3. Go to Armoury and change Logo setting to something else (for example turn it ON and OFF, etc)

4. Check if Monitoring Studio catching a lot of data (Armoury spams it like crazy). If NOT - you probably have selected wrong Asus device on step 2 (so need to re-do it)

5. In Monitoring Studio select ``Tools`` -> ``Save to Log``, and save data
![Screenshot 2024-01-20 163247](https://github.com/seerge/g-helper/assets/5920850/ad480dcf-05bc-43a7-9a80-2984fb68e8a8)

Upload this data here. 

Thanks in advance.

**Reply by IsoMan909** [reply]
unfortunately, after installing AC again I was surprised that the logo option is no longer available.
I'm sure that it was available in older versions and I disabled it.
now I can't turn it off!

**Reply by seerge** [reply]
@IsoMan909 if AC can't turn it off - i can't help anyhow
